ntoskrnl/include/internal/i386/mm.h

```diff
@@ -128,21 +117,69 @@
 /* On x86, these two are the same */
 #define MI_WRITE_VALID_PPE MI_WRITE_VALID_PTE
 
-/* Convert an address to a corresponding PTE */
-#define MiAddressToPte(x) \
-    ((PMMPTE)(((((ULONG)(x)) >> 12) << 2) + PTE_BASE))
+/*  Translating virtual addresses to physical addresses
+        (See: "Intel� 64 and IA-32 Architectures Software Developer�s Manual
+              Volume 3A: System Programming Guide, Part 1, CHAPTER 4 PAGING")
+    Page directory (PD) and Page table (PT) definitions
+    Page directory entry (PDE) and Page table entry (PTE) definitions
+*/
+
+/* Maximum number of page directories pages */
+#ifndef _X86PAE_
+#define PD_COUNT 1        /* Only one page directory page */
```

Comment:
Please don't add these back! I'm very happy that they were removed. There is PDE_PER_PAGE, and that is an "official" definition. This one is pointless, since it just a different and less useful definition of PDE_PER_PAGE.